pages/api/tags/delete-tag.js

```diff
@@ -0,0 +1,82 @@
+import * as Utilities from "~/node_common/utilities";
+import * as Data from "~/node_common/data";
+import * as ViewerManager from "~/node_common/managers/viewer";
+import * as SearchManager from "~/node_common/managers/search";
+
+export default async (req, res) => {
+  const id = Utilities.getIdFromCookie(req);
+  if (!id) {
+    return res.status(500).send({ decorator: "SERVER_GET_SLATES_TAGS", error: true });
+  }
+
+  const user = await Data.getUserById({
+    id,
+  });
+
+  if (!user) {
+    return res.status(404).send({
+      decorator: "SERVER_GET_SLATE_USER_NOT_FOUND",
```

Comment:
And all other decorators will be "SERVER" + function name + error name 
Here it would be something like "SERVER_GET_USER_TAGS_USER_NOT_FOUND"
And "SERVER_GET_USER_TAGS_SLATES_NOT_FOUND" for when you are trying to getSlatesByUserId below

For every new error decorator, we then go into messages.js and add a coresponding error message to show the user (this is the message that appears in the red alert on the page when something goes wrong)

Reply:
Totally don't expect you to know this already since it isn't in the codebase yet (it's something I'm standardizing during the migration), but just wanted to give you a heads up about the upcoming change

Reply:
Thank you. I am going to add the corresponding messages